### PR TITLE
Clean up:

### PR DESCRIPTION
**Pull request by jacobweinstock**
## Description

Remove references to sandbox. Update versions for helm, k3d, etc. Update template images to use latest as quay.io/tinkerbell/actions/ don't have tags.

## Why is this needed

Fixes: #

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

File: vagrant/.env [hunk 1]
```diff
@@ -6,5 +6,7 @@ MACHINE1_MAC=08:00:27:9e:f5:3a
 
 # https://github.com/tinkerbell/charts/pkgs/container/charts%2Fstack
 HELM_CHART_VERSION=0.4.3
-KUBECTL_VERSION=1.28.3
+KUBECTL_VERSION=1.27.12
```
Comment (chrisdoherty4) [inline vagrant/.env:9]: We're going backwards?

Comment (chrisdoherty4) [reply vagrant/.env:9]: Should we even have a `.env` here? Should it be `.env.example`?

Comment (jacobweinstock) [reply vagrant/.env:9]: yeah, because the k3d version is actually 1.27.

Comment (jacobweinstock) [reply vagrant/.env:9]: the `.env` is not an example. its used when we run `vagrant up`.

Comment (chrisdoherty4) [reply vagrant/.env:9]: Right, but do we expect it to change from 1 env to the other? That's typically how .env is used.

**Review by chrisdoherty4 — APPROVED** [review]

File: vagrant/.env (hunk 1 above)
Comment (jacobweinstock) [reply vagrant/.env:9]: yes and no. users can change these values but the name doesnt have any particular significance. its just a file to hold values. We could change it.

**Pull request merged**